src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterEnrichments.scala

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import org.broadinstitute.dsde.workbench.leonardo.model.Cluster
+import org.scalactic.Equality
+
+object ClusterEnrichments {
+  // When in scope, Equality instances override Scalatest's default equality ignoring the id field
+  // while comparing clusters as we typically don't care about the database assigned id field
+  // http://www.scalactic.org/user_guide/CustomEquality
+  implicit val clusterEq = {
+    new Equality[Cluster] {
+      private val FixedId = 0
+
+      def areEqual(a: Cluster, b: Any): Boolean = {
+        b match {
+          case c: Cluster => a.copy(id = FixedId) == c.copy(id = FixedId)
+          case _ => false
+        }
+      }
+    }
+  }
+
+  implicit val clusterSeqEq = {
```

Comment:
Ick, does scalatest make you define an `Equality` for `Set[A]` and `Seq[A]`, even if there is an implicit `Equality[A]` in scope?

Wouldn't be necessary with cats `Eq` :)

Reply:
(Not suggesting we use cats as this is definitely scalatest-specific -- just complaining :))

Reply:
I know, it's saddening.